### DISCUSSION TITLE
Release 2.2.1

### DISCUSSION THREAD

**Discussion by androidacy-user** [Androidacy/MagiskModuleManager#49] [General]
## What's Changed
[Read more](https://www.androidacy.com/androidacy-module-manager-amm-2-2-1-release/#utm_source=GitHub&utm_medium=GitHub&utm_campaign=github-release-211)

**NOTE: This will be the last release with APKs uploaded to GitHub. Further releases will be exclusive to the website, which we encourage you to download this release from as well if possible. We'll be opening a form to report download issues soon for the few people still having them.**

<hr /><em>This discussion was created from the release <a href='https://github.com/Androidacy/MagiskModuleManager/releases/tag/2.2.1'>Release 2.2.1</a>.</em>

**Comment by now0clock** [comment]
Seeing as how the last discussion went, I will be uninstalling this application ASAP. It was interesting. Peace.

**Comment by androidacy-user** [comment]
> Seeing as how the last discussion went, I will be uninstalling this application ASAP. It was interesting. Peace.

You do not have to like our decisions, we're not changing our business model because of a few users who demand everything for nothing and frankly I tire of that expectation and I know that other developers in this community are tired too.

There is a constant expectation from this community to give our everything in relation to development, support, and services and get absolutely nothing in return and be happy with it. Well, I'm sorry, but bills gotta get paid and you threatening to uninstall the app is reminiscent of a Karen in a grocery store.

The fact is the majority of our userbase have no problems with how we run things, and our model works well for us.

Peace.
